src/PheanstalkLocator.php

```diff
@@ -36,6 +37,11 @@ public function getPheanstalk(string $name = null): ?PheanstalkInterface
         return null;
     }
 
+    public function getPheanstalkName(PheanstalkInterface $pheanstalk): ?string
+    {
+        return array_search($pheanstalk, $this->pheanstalks, true);
```

Comment:
`array_search()` returns `false` not `null`, if a value is not found.
```
return array_search($pheanstalk, $this->pheanstalks, true) ?: null;
```
Actually this approach would fail with a pheanstalk named `'0'`, so we have to check for `false`.
```
$name = array_search($pheanstalk, $this->pheanstalks, true);

return false === $name ? null : $name;
```

Reply:
Good point.